pkg/util/test/util.go

```diff
@@ -178,6 +178,7 @@ var (
 	opVer23_5_1, _                    = version.NewVersion("23.5.1-")
 	opVer23_7, _                      = version.NewVersion("23.7.0-")
 	minOpVersionForKubeSchedConfig, _ = version.NewVersion("1.10.2-")
+	pxOperatorMasterVersion, _        = version.NewVersion("99.9.9")
```

Comment:
Can we use this `PxOperatorMasterVersion` instead of "99.9.9", its already defined, just in case we change it to something else

Reply:
You mean to say `PxOperatorMasterVersion` has already been defined elsewhere?

Reply:
Yes, in same file in const